CMakeLists.txt

```diff
@@ -439,6 +439,33 @@ if (PRECICE_ENABLE_FORTRAN)
   # include((${CMAKE_CURRENT_LIST_DIR}/extras/bindings/fortran/CMakeLists.txt)
   add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/extras/bindings/fortran)
 endif()
+
+# Build benchmarks if enabled
+option(PRECICE_ENABLE_BENCHMARKS "Build the preCICE benchmarks" OFF)
+if(PRECICE_ENABLE_BENCHMARKS)
+  find_package(benchmark REQUIRED)
+
+  add_executable(benchprecice)
```

Comment:
In accordance to other executables around preCICE, I would call this `precice-bench`.  I would even consider merging it optionally with the `precice-tools`

Reply:
Good point. Renaming `testprecice` would also make sense.